google/cloud/forseti/notifier/notifiers/email_violations.py

```diff
@@ -52,8 +53,13 @@ def __init__(self, resource, inventory_index_id,
                                               global_configs,
                                               notifier_config,
                                               notification_config)
-        self.mail_util = email.EmailUtil(
-            self.notification_config['sendgrid_api_key'])
+        if self.notifier_config.get('email_connector'):
+            self.connector = email_factory.EmailFactory(
```

Comment:
An exception might be raised from the EmailFactory, why is it not handled here?

Reply:
Handled potential exception.